modules/data_integrity_flag/js/dataIntegrityFlagIndex.js

```diff
@@ -0,0 +1,2 @@
"function")?call:self}function _inherits(subClass,superClass){if(typeof superClass!=="function"&&superClass!==null){throw new TypeError("Super expression must either be null or a function, not "+(typeof superClass==="undefined"?"undefined":_typeof(superClass)))}subClass.prototype=Object.create(superClass&&superClass.prototype,{constructor:{value:subClass,enumerable:false,writable:true,configurable:true}});if(superClass)Object.setPrototypeOf?Object.setPrototypeOf(subClass,superClass):subClass.__proto__=superClass}var SetFlagForm=function(_Component){_inherits(SetFlagForm,_Component);function SetFlagForm(props){_classCallCheck(this,SetFlagForm);var _this=_possibleConstructorReturn(this,(SetFlagForm.__proto__||Object.getPrototypeOf(SetFlagForm)).call(this,props));_this.state={formData:{}};_this.setFormData=_this.setFormData.bind(_this);_this.handleSubmit=_this.handleSubmit.bind(_this);return _this}_createClass(SetFlagForm,[{key:"setFormData",value:function setFormData(formElement,value){var formData=JSON.parse(JSON.stringify(this.state.formData));formData[formElement]=value;this.setState({formData})}},{key:"handleSubmit",value:function handleSubmit(e){e.preventDefault();$.ajax({type:"POST",url:"/data_integrity_flag/",data:JSON.stringify(this.state.formData),cache:false,contentType:false,processData:false,success:function(data){swal({title:"Success!",type:"success"});this.props.updateData();this.setState({formData:{}})}.bind(this),error:function error(err){console.error(err);swal({title:"Error!",type:"error",content:err.statusText})}})}},{key:"render",value:function render(){var fieldOptions=this.props.fieldOptions;return _react2.default.createElement("div",{className:"col-md-8 col-lg-6"},_react2.default.createElement(FormElement,{name:"flag_form",onSubmit:this.handleSubmit},_react2.default.createElement("h3",{className:"text-center"},"Update Instrument Status"),_react2.default.createElement("br",null),_react2.default.createElement(SelectElement,{name:"visitLabel",label:"Visit Label",options:fieldOptions.visits,onUserInput:this.setFormData,value:this.state.formData.visitLabel,required:true}),_react2.default.createElement(SelectElement,{name:"instrument",label:"Instrument",options:fieldOptions.instruments,onUserInput:this.setFormData,value:this.state.formData.instrument,required:true}),_react2.default.createElement(DateElement,{name:"date",label:"Date",onUserInput:this.setFormData,value:this.state.formData.date,required:true}),_react2.default.createElement(SelectElement,{name:"flagStatus",label:"Flag Status",options:fieldOptions.flagStatusList,onUserInput:this.setFormData,value:this.state.formData.flagStatus,required:true}),_react2.default.createElement(TextareaElement,{name:"comment",label:"Comment",onUserInput:this.setFormData,value:this.state.formData.comment}),_react2.default.createElement(ButtonElement,{label:"Update"})))}}]);return SetFlagForm}(_react.Component);SetFlagForm.propTypes={};SetFlagForm.defaultProps={};exports.default=SetFlagForm}]);
```

Comment:
can be deleted because on major

Reply:
@HenriRabalais this still needs to be deleted along with the compiled index.js

Reply:
@done